scripts/urban.js

```diff
@@ -28,23 +30,23 @@ module.exports = function (robot) {
 
         var udResp = JSON.parse(body); // The urban dictionary response, parsed into a JSON object.
 
-        // Parse Urban Dictionary response and send result.
-        if (udResp["result_type"] === "exact") {
-          var firstResult = udResp["list"][0];
-          var definition = firstResult["definition"];
-          var example = firstResult["example"];
-          var response = phrase.toUpperCase() + ":\n" + definition.toString() + "\n";
-          if (example) {
-            response += ">>> " + example.toString();
-          }
-          res.send(response);
-          if (udResp["list"].length > 1) {
-            res.send(" - more definitions at http://www.urbandictionary.com/define.php?term=" + encodeURI(phrase));
-          }
-          return;
-        } else {
+        // Check that a result was found.
+        if (udResp["result_type"] !== 'exact') {
           return res.send("> No results found for " + phrase +". ¯\\_(ツ)_/¯")
         }
+
+        // Parse Urban Dictionary response and send result.
+        var firstResult = udResp["list"][0];
+        var definition = firstResult["definition"];
+        var example = firstResult["example"];
+        var response = phrase.toUpperCase() + ":\n" + definition.toString() + "\n";
+        if (example) {
+          response += ">>> " + example.toString();
+        }
+        res.send(response);
+        if (udResp["list"].length > 1) {
```

Comment:
I would put this a bit higher and `+=` it onto the response so you can send it as one message :^)